test/integration/targets/docker_container/tasks/tests/network.yml

```diff
@@ -535,6 +553,172 @@
 
   when: docker_py_version is version('1.10.0', '>=')
 
+####################################################################
+## networks with IP address ########################################
+####################################################################
+
+- block:
+  - set_fact:
+      nname_3_ipv4_2: "{{ subnet_ipv4 | next_nth_usable(2) }}"
+      nname_3_ipv4_3: "{{ subnet_ipv4 | next_nth_usable(3) }}"
+      nname_3_ipv4_4: "{{ subnet_ipv4 | next_nth_usable(4) }}"
+      nname_3_ipv6_2: "{{ subnet_ipv6 | next_nth_usable(2) }}"
+      nname_3_ipv6_3: "{{ subnet_ipv6 | next_nth_usable(3) }}"
+      nname_3_ipv6_4: "{{ subnet_ipv6 | next_nth_usable(4) }}"
+  - debug:
+      msg: "{{ nname_3_ipv4_2 }} {{ nname_3_ipv4_3 }} {{ nname_3_ipv4_4 }} {{ nname_3_ipv6_2 }} {{ nname_3_ipv6_3 }} {{ nname_3_ipv6_4 }}"
+
+  - name: create container (stopped) with one network and fixed IP
+    docker_container:
+      image: alpine:3.8
+      command: '/bin/sh -c "sleep 10m"'
+      name: "{{ cname }}"
+      state: stopped
+      networks:
+      - name: "{{ nname_3 }}"
+        ipv4_address: "{{ nname_3_ipv4_2 }}"
+        ipv6_address: "{{ nname_3_ipv6_2 }}"
+      networks_cli_compatible: yes
```

Comment:
Does `networks_cli_compatible` need to be enabled for these checks ?

Reply:
No, but `networks_cli_compatible: yes` will be the default eventually anyway (the default will switch in Ansible 2.12, right now you get a warning if you don't explicitly set it to `yes` or `no` when specifying `networks:`).

Reply:
If there is no reason to not use the default value, i would remove this parameter.

Reply:
All tests involving `networks:` which don't explicitly test with `networks_cli_compatible: no` use `networks_cli_compatible: yes`, which we recommend all users to use when they specify `networks:`. So I'd really prefer this test to also use the new codepaths and not the old ones corresponding to the old deprecated default value.